### PR TITLE
Optimize the description of disabling swap

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Optimize the description of disabling swap

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by AstroProfundis — APPROVED** [review]

File: check-before-deployment.md [hunk 1]
```diff
@@ -108,9 +108,14 @@ aliases: ['/docs-cn/dev/check-before-deployment/']
 
 ## 检测及关闭系统 swap
 
-本段介绍 swap 关闭方法。TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能。因此建议永久关闭系统 swap，并且不要使用 `swapoff -a` 方式关闭，否则重启机器后该操作会失效。
+不建议将 swap 用作内存，因为这会降低性能，建议永久关闭系统 swap。
```
Comment (TomShawn) [inline check-before-deployment.md:111]: 感觉原来的说明很清晰，改后的描述显得很简略不清晰？

File: check-before-deployment.md [hunk 2]
```diff
@@ -108,9 +108,14 @@ aliases: ['/docs-cn/dev/check-before-deployment/']
...
 
-建议执行以下命令关闭系统 swap：
+关闭 swap:
```
Comment (TomShawn) [inline check-before-deployment.md:113]:
```suggestion
要关闭 swap，可执行以下步骤:
```

File: check-before-deployment.md [hunk 3]
```diff
@@ -108,9 +108,14 @@ aliases: ['/docs-cn/dev/check-before-deployment/']
...
+
+1. 设置内核参数 `swappiness` 为 `0`, 即不使用 swap；
```
Comment (TomShawn) [inline check-before-deployment.md:115]: 建议不使用分号作为 item 结尾
```suggestion
1. 设置内核参数 `swappiness` 为 `0`，即不使用 swap。
```

File: check-before-deployment.md [hunk 4]
```diff
@@ -108,9 +108,14 @@ aliases: ['/docs-cn/dev/check-before-deployment/']
...
+2. 应用新的内核参数；
+3. 执行 `swapoff -a`，使已使用的 swap 清空；
+4. 执行 `swapon -a`，此时，由于 `swappiness` 为 `0`，实际不会使用 swap。
 
 {{< copyable "shell-regular" >}}
```
Comment (TomShawn) [inline check-before-deployment.md:120]: 加了前面的步骤后，这些命令是否需要调整或加上说明呢？

File: check-before-deployment.md (hunk 1 above)
Comment (shichun-0415) [reply check-before-deployment.md:111]:
```suggestion
TiDB 运行需要有足够的内存，但不建议使用 swap 作为内存不足的缓冲，因为这会降低性能，建议永久关闭系统 swap。
```

Comment (shichun-0415) [reply check-before-deployment.md:111]: @TomShawn PTAL. Thanks.

File: check-before-deployment.md [hunk 5]
```diff
@@ -108,9 +108,14 @@ aliases: ['/docs-cn/dev/check-before-deployment/']
...
-本段介绍 swap 关闭方法。TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能。因此建议永久关闭系统 swap，并且不要使用 `swapoff -a` 方式关闭，否则重启机器后该操作会失效。
+不建议将 swap 用作内存，因为这会降低性能，建议永久关闭系统 swap。
 
-建议执行以下命令关闭系统 swap：
+要关闭 swap，可执行以下步骤:
+
+1. 设置内核参数 `swappiness` 为 `0`，即不使用 swap。
+2. 应用新的内核参数；
+3. 执行 `swapoff -a`，使已使用的 swap 清空；
+4. 执行 `swapon -a`，此时，由于 `swappiness` 为 `0`，实际不会使用 swap。
```
Comment (shichun-0415) [inline check-before-deployment.md:118]:
```suggestion
1. 设置内核参数 `swappiness` 为 `0`，即不使用 swap。
2. 应用新的内核参数。
3. 执行 `swapoff -a`，使已使用的 swap 清空。
4. 执行 `swapon -a`，此时，由于 `swappiness` 为 `0`，实际不会使用 swap。
```

File: check-before-deployment.md [hunk 6]
```diff
@@ -120,6 +120,12 @@ swapoff -a && swapon -a
 sysctl -p
 ```
 
+> **注意：**
+>
+> - 执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。如果只执行 `swapoff`，关闭 swap 的命令会在重启后失效。
```
Comment (shichun-0415) [inline check-before-deployment.md:125]:
```suggestion
> - 执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。如果只执行 `swapoff`，重启后 swap 会再次自动打开。
```

File: check-before-deployment.md [hunk 7]
```diff
@@ -120,6 +120,12 @@ swapoff -a && swapon -a
 sysctl -p
 ```
 
+> **注意：**
+>
+> - 执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。如果只执行 `swapoff`，重启后 swap 会再次自动打开。
```
Comment (shichun-0415) [inline check-before-deployment.md:125]:
```suggestion
> - 执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。如果只执行 `swapoff -a`，重启后 swap 会再次自动打开。
```

File: check-before-deployment.md [hunk 8]
```diff
@@ -108,9 +108,9 @@ aliases: ['/docs-cn/dev/check-before-deployment/']
 
 ## 检测及关闭系统 swap
 
-本段介绍 swap 关闭方法。TiDB 运行需要有足够的内存，并且不建议使用 swap 作为内存不足的缓冲，这会降低性能。因此建议永久关闭系统 swap，并且不要使用 `swapoff -a` 方式关闭，否则重启机器后该操作会失效。
+TiDB 运行需要有足够的内存，但不建议使用 swap 作为内存不足的缓冲，因为这会降低性能，建议永久关闭系统 swap。
```
Comment (TomShawn) [inline check-before-deployment.md:111]:
```suggestion
TiDB 运行需要有足够的内存。如果内存不足，不建议使用 swap 作为内存不足的缓冲，因为这会降低性能。建议永久关闭系统 swap。
```

File: check-before-deployment.md [hunk 9]
```diff
@@ -120,6 +120,12 @@ swapoff -a && swapon -a
 sysctl -p
 ```
 
+> **注意：**
+>
+> - 执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。如果只执行 `swapoff -a`，重启后 swap 会再次自动打开。
+> - 执行 `sysctl -p` 命令是为了在不重启的情况下使配置生效。
+>
```
Comment (TomShawn) [inline check-before-deployment.md:127]:
```suggestion
```

File: check-before-deployment.md [hunk 10]
```diff
@@ -120,6 +120,12 @@ swapoff -a && swapon -a
 sysctl -p
 ```
 
+> **注意：**
+>
+> - 执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。如果只执行 `swapoff -a`，重启后 swap 会再次自动打开。
```
Comment (TomShawn) [inline check-before-deployment.md:125]:
```suggestion
> - 一起执行 `swapoff -a` 和 `swapon -a` 命令是为了刷新 swap，将 swap 里的数据转储回内存，并清空 swap 里的数据。不可只执行 `swapoff -a` 而省略 `swapon -a`，否则重启后 swap 会再次自动打开，使得操作失效。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**